src/Hud/XpRate/XpRatePlugin.cs

```diff
@@ -108,8 +108,14 @@ public override void Render()
                         if (dif < 0) { bounds.X += dif; bounds.Width -= dif; }
 
                         Graphics.DrawText(timer, Settings.TextSize, new Vector2(bounds.X + 70, position.Y), Settings.TimerTextColor);
-                        Graphics.DrawText(fps, Settings.TextSize, new Vector2(bounds.X + 70, secondLine.Y), Settings.FpsTextColor);
-                        Graphics.DrawText(ping, Settings.TextSize, new Vector2(bounds.X + 70, thirdLine.Y), Settings.LatencyTextColor);
+                        if (Settings.ShowFps)
+                        {
+                            Graphics.DrawText(fps, Settings.TextSize, new Vector2(bounds.X + 70, secondLine.Y), Settings.FpsTextColor);
+                        }
+                        if (Settings.ShowLatency)
```

Comment:
hmm, well that also doesn't make any sense cuz when you enable "Only Area Name" you cannot differentiate between FPS number and latency number.... So this PR doesn't is invalid, other than following two lines where you fix the bug

`Size2 xpRateSize = Graphics.DrawText(timeLeft, Settings.TextSize, secondLine, Settings.TimeLeftColor, FontDrawFlags.Right);`
`Size2 xpLeftSize = Graphics.DrawText(xpReceivingText, Settings.TextSize, thirdLine, Settings.XphTextColor, FontDrawFlags.Right);`